frontend-plugin-core/src/main/java/com/github/eirslett/maven/plugins/frontend/lib/IncrementalMojoHelper.java

```diff
@@ -134,8 +143,13 @@ public boolean canBeSkipped(String arguments, ExecutionCoordinates coordinates,
                         }
                     }
                 }
+
+                startTimeForSaveTimeUpdate = Optional.of(now());
+            } else {
+                startTimeForSaveTimeUpdate = Optional.of(now());
             }
```

Comment:
Nit: seems like it always sets it?
```suggestion
            }
            startTimeForSaveTimeUpdate = Optional.of(now());
```

Reply:
Ah, it should've been in the else above. I've restructured it to make it fix and make it less prone to this bug. There's only one execution path to incremental so we can clear the time tracking there.